v2.4/getting-started/kubernetes/tutorials/advanced-policy.md

```diff
@@ -226,18 +203,17 @@ EOF
 Notice that we've specified an order of 500.  This means that this policy will be applied before any
 of the Kubernetes policies.
 
-We also need to create a policy which allows traffic to access kube-dns.  Let's create a policy which
-applies to the entire cluster, and allows it to access kube-dns pods in the kube-system Namespace.
+We also need to create a policy which allows traffic to access kube-dns.  Let's create a policy that allows it to access kube-dns pods in the kube-system Namespace.
```

Comment:
Sentence is a bit redundant. How about 

`We also need to create a policy which allows traffic to access kube-dns.  Let's create one now in the kube-system Namespace.`